schema/spacewalk/common/tables/rhnActionVirtCreate.sql

```diff
@@ -34,7 +34,8 @@ CREATE TABLE rhnActionVirtCreate
     cobbler_system       VARCHAR(256),
     kickstart_host       VARCHAR(256),
     kernel_options       VARCHAR(2048),
-    cluster_definitions  VARCHAR(2048)
+    cluster_definitions  VARCHAR(2048),
+    template             VARCHAR(20)
```

Comment:
Are all template names below 20 characters ?

Reply:
Nothing prevents them to have longer names: those are coming from `virt-tuner` itself.

Reply:
Since I merged the create action refactoring first, this column will just go into the new TEXT column. No need to worry about that after the rebase.